packages/flutter/lib/src/widgets/scroll_view.dart

```diff
@@ -894,7 +894,11 @@ abstract class BoxScrollView extends ScrollView {
 /// ## Transitioning to [CustomScrollView]
 ///
 /// A [ListView] is basically a [CustomScrollView] with a single [SliverList] in
-/// its [CustomScrollView.slivers] property.
+/// its [CustomScrollView.slivers] property. Instead of using [shrinkWrap], prefer
```

Comment:
this sentence is odd here because in the example that follows both the CusomScrollView and the ListView have `shrinkWrap` set. I believe the video talks about a different scenario than the one that's talked about in this section...